src/index.html

```diff
@@ -13,5 +13,178 @@
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset_form">
```

Comment:
```suggestion
      <fieldset class="fieldset-form">
```
better to use `-` instead `_`

Reply:
fixed